### PR TITLE
[15.05] Fix workflow parameter/action display

### PR DESCRIPTION
**Pull request by dannon**
Overzealous sanitization in c8a71c7e8988d80bf406f11bfb9b8d3ed815e97e.

### CODE REVIEW

File: templates/webapps/galaxy/workflow/run.mako [hunk 1]
```diff
@@ -583,9 +583,10 @@ if wf_parms:
                     <%
                     pja_ss_all = []
                     for pja_ss in [ActionBox.get_short_str(pja) for pja in step.post_job_actions]:
-                        pja_ss = h.escape( pja_ss )
                         for rematch in re.findall('\$\{.+?\}', pja_ss):
```
Comment (jmchilton) [inline templates/webapps/galaxy/workflow/run.mako:586]: I may be missing something but there seem to be a lot of seemingly unescaped values that go into `get_short_str` right?

```
            return "Add tag(s) '%s' to '%s'." % (pja.action_arguments['tags'], pja.output_name)
```

```
        return "Set the datatype of output '%s' to '%s'" % (pja.output_name, pja.action_arguments['newtype'])
```

```
            return "Rename output '%s' to '%s'." % (pja.output_name, pja.action_arguments['newname'])
```

```
        return "Hide output '%s'." % pja.output_name
```

Or are these things escaped going into the pja?

If I am not wrong - I would be happy to take a crack at pushing the escape stuff into the short string also so those things can be escaped without escaping the whole thing.

Comment (dannon) [inline templates/webapps/galaxy/workflow/run.mako:586]: Good points, it'd be easier to sanitize at get_short_str and trust that output.  I'll update the PR.

Comment (dannon) [inline templates/webapps/galaxy/workflow/run.mako:586]: Now that I poke at it -- in your examples you're not trusting tool.output_name.  Intentional?  If so, makes me wonder where else we're either trusting it and shouldn't be, or where it's being unnecessarily escaped.  Action arguments should definitely be escaped as these are arbitrary user input.

**Comment by jmchilton** [comment]
:+1: - Thanks for the fix @dannon.

**Pull request merged**